src/helpers/config.ts

```diff
@@ -0,0 +1,10 @@
+// only for internal usage
+
+import { appConfigInit, CoreNetworkName } from './appConfig';
+
+require('dotenv').config();
```

Comment:
for a package being used as effect-free library, would recommend to not use dotenv.config() it's the opposite of being effect-free.

Reply:
caching will not be included in the package, it will break the UI

Reply:
But there#s other ways for setting env then using effect-ful dotenv (e.g. by just forcing env management on userland or script level).

Just saying with this the thing is not effect-free as the package.json suggests.

Reply:
What's the problem with using `dotenv `?

Reply:
there's no problem with dotenv itself (we use it everywhere).
My problem is just the usage here, as it makes the code non effect-free.
Also logic wise, imo a utility `library` should not handle env loading (as it should and probably already is done, in the package using it as a dependency.

Reply:
this is used only internally so that the GitHub runner starts caching for all governance cores (mainnet and testnets (sepolia, goerli)) at once, and it is also done so that I can easily define this in my UI.

comment on the top of this file:
`only for internal usage` - this means that this file is used only for internal use and is not included in the package

therefore, further, if you looked at the code, this is used only for the caching system, since it will also not be included in the package

or if it does, then we will assemble the caching system itself in a separate package, since I don’t see any point in getting so many extra code in the UI

and even then I’m ready to return to this issue